owncloudApp/src/main/java/com/owncloud/android/features/FeatureList.java

```diff
@@ -41,26 +41,28 @@
     static final private boolean SHOW_ON_UPGRADE = false;
 
     static final private int VERSION_CODE_BETA_2_15_1 = 21400201;
+    static final private String DEFAULT_WIZARD_VERSION_NAME = "2.7.0";
+    static final private String INDIFFERENT = "0";
     static final private FeatureItem INCONSISTENT_VERSION = new FeatureItem(
             R.drawable.whats_new_warning,
             R.string.welcome_feature_6_title,
             R.string.welcome_feature_6_text,
-            "2.14.2",
-            "0",
+            INDIFFERENT,
+            INDIFFERENT,
```

Comment:
The first one shouldn't be "0", right?

Reply:
Doesn't matter since it will be showed when lastSeenVersion is a determined versionName, we don't need to check when it was included.